languages/csharp/exercises/concept/arrays/.docs/instructions.md

```diff
@@ -0,0 +1,57 @@
+In this exercise, you're an avid bird watcher that keeps track of how many birds have visited your garden in the last seven days.
+
+You have five tasks, all dealing with the numbers of birds that visited your garden.
+
+### 1. Calculate the total number of visiting birds
+
+Implement the `BirdCount.Total()` method to return the total number of birds that have visited your garden:
+
+```csharp
+int[] dayCounts = { 2, 5, 1, 4, 7, 0 };
+var birdCount = new BirdCount(dayCounts);
+birdCount.Total();
+// => 19
+```
+
+### 2. Calculate the number of busy days
+
+Some days are busier that others. A busy day is one where five or more birds have visited your garden.
+Implement the `BirdCount.BusyDays()` method to return the number of busy days:
+
+```csharp
+int[] dayCounts = { 2, 5, 1, 4, 7, 0 };
+var birdCount = new BirdCount(dayCounts);
+birdCount.BusyDays();
+// => 2
+```
+
+### 3. Check how many birds visited yesterday
+
+Implement the `BirdCount.Yesterday()` method to return how many birds visited your garden yesterday, which is the second last element:
```

Comment:
```suggestion
Implement the `BirdCount.Yesterday()` method to return how many birds visited your garden yesterday, which is the penultimate element:
```

Reply:
Ah. I did some googling and found some things on "second last". I do think "penultimate" will be unfamiliar to many non-native speakers. What about:

```suggestion
Implement the `BirdCount.Yesterday()` method to return how many birds visited your garden yesterday, which is the sixth element:
```

Reply:
I think indexing from the right like you had is better. How about "last but one"?

Reply:
This is a tricky one to describe in this manner to non-native speakers (or even native speakers from different regions). Also, reading until this point I hadn't considered how the ordering would related to the current day. Perhaps it could be explained earlier in the text that the array is ordered so that today's count is the last, or right-most, count in the array?

Reply:
Yes, let me explain the ordering a bit. I'll rephrase.